src/components/label/label.scss

```diff
@@ -101,9 +101,3 @@
     @apply bg-opacity-0;
   }
 }
-
-:host([disable-spacing]) {
-  .container {
-    @apply m-0 gap-0;
```

Comment:
@macandcheese @asangma do we need another CSS var for setting gap to zero?  `--calcite-label-margin-bottom` only sets the margin so a gap still exists. Could we just have one or another somehow?

Reply:
ping @eriklharper @macandcheese do we need both gap and margin? If so, should I introduce another variable for devs to remove the gap?